zeeguu_api/app.py

```diff
@@ -25,7 +25,7 @@
 
 try:
     import dashboard
-    dashboard.config.from_file('/home/mircea/zee/http/api/dashboard.cfg')
+    dashboard.config.from_file('/home/travis/build/zeeguu-ecosystem/Zeeguu-API/dashboard_test.cfg')
```

Comment:
hmm. instead of fighting for this string, maybe it's good to use an envvar for the location of the dashboard.cfg? e.g. DASHBOARD_CFG ?

Reply:
Yes, this would be better. I can implement this such that the Dashboard will take the envvar value when specified, and otherwise takes the dashboard.config.from_file().